<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JigsawRunWidget</class>
 <widget class="QDockWidget" name="JigsawRunWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>88</width>
    <height>107</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>&amp;Jigsaw</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Jigsaw</string>
   </property>
   <property name="windowIcon">
    <iconset>
     <normaloff>icons/jigsaw.png</normaloff>icons/jigsaw.png</iconset>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>618</width>
         <height>547</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="statusLabel">
              <property name="text">
               <string>Status:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="statusOutputLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="iterationLabel">
              <property name="text">
               <string>Iteration</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="iterationLcdNumber">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="pointLabel">
              <property name="text">
               <string>Point</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="pointLcdNumber">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>   Control Network   </string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout">
               <item row="2" column="1">
                <widget class="QLCDNumber" name="measuresLcdNumber">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLCDNumber" name="pointsLcdNumber">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLCDNumber" name="imagesLcdNumber">
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" alignment="Qt::AlignRight">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Images</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" alignment="Qt::AlignRight">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Measures</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" alignment="Qt::AlignRight">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Points</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="rmsAdjustedPointSigmasGroupBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="title">
               <string>RMS Adj Point Sigmas</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="2" column="0" alignment="Qt::AlignRight">
                <widget class="QLabel" name="radiusLcdLabel">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Radius</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" alignment="Qt::AlignRight">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Latitude</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" alignment="Qt::AlignRight">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Longitude</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLCDNumber" name="latitudeLcdNumber">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="smallDecimalPoint">
                  <bool>false</bool>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLCDNumber" name="longitudeLcdNumber">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLCDNumber" name="radiusLcdNumber">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
              <zorder>label_4</zorder>
              <zorder>label_7</zorder>
              <zorder>radiusLcdLabel</zorder>
              <zorder>latitudeLcdNumber</zorder>
              <zorder>longitudeLcdNumber</zorder>
              <zorder>radiusLcdNumber</zorder>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="detachedLabelsCheckBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Write Detached Image Labels on Accept</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="useLastSettings">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Use Last Accepted Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="JigsawSetupButton">
                <property name="text">
                 <string>&amp;Setup</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="JigsawRunButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>&amp;Run</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="JigsawAcceptButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Accept</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>18</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QScrollArea" name="statusUpdateScrollArea">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QLabel" name="statusUpdatesLabel">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>594</width>
               <height>151</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>1</horstretch>
               <verstretch>2</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Welcome to Jigsaw</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
